src/utils/filterPeople.ts

```diff
@@ -0,0 +1,48 @@
+import { GenderFilters, Person, SearchQueries } from '../types';
+
+export function filterPeople(
+  people: Person[],
+  params: URLSearchParams,
+): Person[] {
+  let filtered = [...people];
+
+  if (params.has(SearchQueries.sex)) {
+    const curParamSex = params.get(SearchQueries.sex);
+
+    if (curParamSex !== GenderFilters.All) {
+      filtered = filtered.filter(person => person.sex === curParamSex);
+    }
+  }
```

Comment:
You're checking if the `params` has `SearchQueries.sex` and then getting its value. Instead, you can directly get the value and check if it's not null. This would save an extra call to `params.has`. Same can be applied for other similar checks.